js/app.js

```diff
@@ -82,6 +117,11 @@
    *   and passing two aditional arguments, event and 'light-green'
    */
 
+  let btn5Args = [event, "light-green"];
```

Comment:
Same, don't reference `event` outside of callback

Reply:
Noted. Is moving the variable declaration inside the event listener the solution?

```javascript
btn5.addEventListener("click", (event) => {
    let btn5Args = [event, "light-green"];
    setMyLightClass.apply(btn5, btn5Args);
  })
```